source/auth.rst

```diff
@@ -3,8 +3,8 @@
 .. _`FedCloud Check-in client`: https://aai.egi.eu/fedcloud
 .. _`AppDB`: https://appdb.egi.eu
 
-Authentication
-==============
+Gettin Access
```

Comment:
Gettin is probably not what should be used.

Reply:
Slang :) doing thing fast has issues

Reply:
:)